### ISSUE TITLE
[Feature Request] Ability to Change About Information

### ISSUE THREAD

**Issue by garv-shah** [jagrosh/MusicBot#886]
**Is your feature request related to a problem? Please describe.**
When running the about command, the bot currently gives a preset answer that I do not believe can be changed.

**Describe the solution you'd like**
It'd be cool if there was a config option to be able to change the text in the about box

**Describe the basic flow/steps of using this feature**
How this feature would be used:
1. Open up config.txt
2. Go to the setting for the about command, which has a json formatted discord embed or the likes
3. Edit the embed

**Additional context**
N/A

**Please check the following links to confirm that your feature has not already been requested:**
https://github.com/jagrosh/MusicBot/labels/enhancement
https://github.com/jagrosh/MusicBot/wiki/Things-That-Won%27t-Be-Added

**Comment by Zelongdragon** [comment]
You can go to the app development page on discord and change the about text there.

**Comment by garv-shah** [comment]
I meant like the about command in the discord server, which returns the discord embed. I managed to edit it by rebuilding the code, but since it uses the example AboutCommand from [here](https://github.com/JDA-Applications/JDA-Utilities/tree/master/examples), it's also relatively hard to change as it's from an external library. It might be cool if [this](https://github.com/JDA-Applications/JDA-Utilities/blob/master/examples/src/main/java/com/jagrosh/jdautilities/examples/command/AboutCommand.java) was edited so it could take in some parameters from the config file :)
<br>
here's what an edited version of the about command would look like
<img width="621" alt="Screen Shot 2021-09-21 at 2 32 27 pm" src="https://user-images.githubusercontent.com/64292655/134111935-d7965f0d-d75d-492c-be35-3dbf37725a71.png">

**Comment by jagrosh** [comment]
I don't think this embed will ever be fully editable, but it may be possible at some point to add an additional section.
